index.html

```diff
@@ -4,170 +4,212 @@
+            />
+          </div>
+          <div class="work-col-2">
+            <h3 class="project-title">Tonic</h3>
+            <div class="frame">
+              <h4 class="client">CANOPY</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
+              <h4 class="role">Back End Dev</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
 
-            <p class="year">2015</p>
+              <p class="year">2015</p>
+            </div>
+            <div class="primary-text">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="tags">
+              <h3>HTML</h3>
+              <h3>CSS</h3>
+              <h3>Ruby on Rails</h3>
+              <h3>JavaScript</h3>
+            </div>
+            <button class="simple-button" type="button">See Project</button>
           </div>
-          <div class="primary-text">
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
+        </div>
+        <div class="card">
+          <div class="work-col-2">
+            <h3 class="project-title">Uber Navigation</h3>
+            <div class="frame">
+              <h4 class="client">Uber</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
+              <h4 class="role">Lead Developer</h4>
+              <img src="./images/Counter.png" alt="" class="counter" />
+
+              <p class="year">2015</p>
+            </div>
+            <div class="primary-text">
+              <p>
+                A smart assistant to make driving more safe, efficient, and fun
+                by unlocking your most expensive computer: your car.
+              </p>
+            </div>
+            <div class="tags">
+              <h3>HTML</h3>
+              <h3>CSS</h3>
+              <h3>Ruby on Rails</h3>
+              <h3>JavaScript</h3>
+            </div>
+            <button class="simple-button" type="button">See Project</button>
           </div>
-          <div class="tags">
-            <h3>HTML</h3>
-            <h3>CSS</h3>
-            <h3>JavaScript</h3>
+          <div class="work-col-1">
+            <img class="snapshoot" src="./images/uber.png" alt="" />
           </div>
-          <button class="simple-button">See Project</button>
         </div>
-        <!-- end of single card -->
-      </div>
-    </section>
+      </section>
+
+      <!-- practice about------- -->
 
-    <!--  about section-->
+      <!-- practice about------- -->
 
-    <section class="about">
-      <div class="container grid">
-        <div class="flex">
+      <!-- ----------about section -->
+      <section class="about">
+        <div class="about-myself">
           <div class="about-title">
             <h2>About Myself</h2>
           </div>
```

Comment:
- [ ] As requested by the previous reviewer, please make sure that the icons used for social media links looks similar to the ones in the Figma design 

> * [ ]  Kindly let's stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). In this about-col-1, we need to ensure that the colors meet the Figma design as follows about section: `#091E42`, about description: `#344563`, icons: `#505F79`. This will help us to make our portfolio match the Figma design and therefore more professional.
> 
> Your design 😎 ![image](https://user-images.githubusercontent.com/99394374/229272738-e65d2837-b30a-4d29-b921-c43ca7259662.png)

Reply:
- [ ] As requested by the previous reviewer and since this project is a great chance to practice HTML and CSS, duplicating the same design from the Figma design system is quite vital for this milestone so kindly ensure that you export the social icons from the Figma design and use those assets for the social links icons 👍🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229312023-00f406a1-4854-40c4-8e55-6b61f6ef6be0.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/229312052-a558e637-ef70-4cac-8434-e8fd17475fdf.png)